### DISCUSSION TITLE
Does neorg support changing the section marker?

### DISCUSSION THREAD

**Discussion by demizer** [nvim-neorg/neorg#290] [Q&A] [answered]
I much rather prefer `=` to `*`. Is this configurable?

**Comment by vhyrro** [comment, marked as answer]
Hey! Unfortunately our file format isn't configurable this way. It's fairly reasonable why this is the case: if we made the format tweakable in any way, imagine how difficult it would be for e.g. github to then render Neorg documents! There'd be no consensus on what a file format should be, and therefore a specification for our file format would become worthless.

Also `=` is already utilized for another feature of our format which we call insertions, so you'd get a clash there as well I'm afraid. Hope that answers your question :)

**Reply by demizer** [reply]
Thank you for the reply.
